Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+thoth-analyzer = "*"
+thoth-common = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
Do you depend on 3.6 or shall we test out 3.8?! @fridex

Reply:
Our s2i base for deploying components uses ubi8-py36 that runs Python 3.6. If python versions will not match, the build will fail and Pipenv/micropipenv will error out as the dependency stack was not created for the specific python version.